neurolib/optimize/exploration/exploration.py

```diff
@@ -418,6 +421,99 @@ def loadDfResults(self, filename=None, trajectoryName=None):
+
+    def _get_coords_from_run(self, run_dict, bold=False):
+        """
+        Find coordinates of a single run - time, output and space dimensions.
+
+        :param run_dict: dictionary with run results
+        :type run_dict: dict
+        :param bold: whether to do only BOLD or without BOLD results
+        :type bold: bool
+        :return: dictionary of coordinates for xarray
+        :rtype: dict
+        """
+        run_dict = copy.deepcopy(run_dict)
+        run_dict = self._filt_dict_bold(run_dict, bold=bold)
+        timeDictKey = ""
+        if "t" in run_dict:
+            timeDictKey = "t"
+        else:
+            for k in run_dict:
+                if k.startswith("t"):
+                    timeDictKey = k
+                    logging.info(f"Assuming {k} to be the time axis.")
+                    break
+        assert len(timeDictKey) > 0, "No time array found (starting with t) in model output."
+        t = run_dict[timeDictKey].copy()
+        del run_dict[timeDictKey]
+        return timeDictKey, {
+            "output": list(run_dict.keys()),
+            "space": list(range(next(iter(run_dict.values())).shape[0])),
+            "time": t,
+        }
+
+    def xr(self, bold=False):
+        """
+        Return xr.Dataset from the exploration. The coordinates are in the star
+        notation if applicable. The whole list of affected model parameters by
+        star notated exploration is listed in attributes.
+
+        :param bold: if True, will load and return only BOLD output
+        :type bold: bool
+        """
+        assert self.results is not None, "Run `loadResults()` first to populate the results"
+        assert len(self.results) == len(self.dfResults)
+        # create intrisinsic dims for one run
+        timeDictKey, run_coords = self._get_coords_from_run(self.results[0], bold=bold)
+        dataarrays = []
+        orig_search_coords = pypet.cartesian_product(self.exploreParameters)
+        for runId, run_result in self.results.items():
+            expl_coords = {k: v[runId] for k, v in orig_search_coords.items()}
+            outputs = []
+            run_result = self._filt_dict_bold(run_result, bold=bold)
+            for key, value in run_result.items():
+                if key == timeDictKey:
+                    continue
+                outputs.append(value)
+            data_temp = xr.DataArray(
+                np.stack(outputs), dims=["output", "space", "time"], coords=run_coords, name="exploration"
+            )
+            expand_coords = {}
+            for k, v in expl_coords.items():
+                if isinstance(v, (str, float, int)):
+                    expand_coords[k] = [v]
+                elif isinstance(v, np.ndarray):
+                    if np.unique(v).size == 1:
+                        expand_coords[k] = [float(np.unique(v))]
+                    else:
+                        raise ValueError("Cannot squeeze coordinates")
```

Comment:
Please explain what the user can do to fix this error.

Reply:
I am afraid nothing... This can happen in `neurolib` native models, where some parameters can be arrays (e.g. `ext_x` in FHN brain net) and you wanna explore this parameter.. then, the respective coordinate for `DataArray` would have to be full array, which is not possible... 
User can do: 
* make sure all values in array are the same (then, this is saved with that one value - no problem)
* if she really needs heterogeneous external currents, use `MultiModel` where each node / mass has its own parameter, therefore this is not a problem